### DISCUSSION TITLE
Cookie authorization

### DISCUSSION THREAD

**Discussion by Frintezza1337** [1N3/Sn1per#410] [Q&A] [answered]
How to transfer cookies or login data if the site is available by authorization? I searched on the Internet, but did not find such information. P.S. Thank you for your hard work and for such a wonderful product!

**Comment by 1N3** [comment, marked as answer]
Currently not available, but this is being tracked as a feature request here: https://github.com/1N3/Sn1per/issues/412

**Reply by Frintezza1337** [reply]
Ok thank you
